koku/masu/processor/parquet/ocp_cloud_parquet_report_processor.py

```diff
@@ -227,12 +227,11 @@ def get_ocp_provider_uuids_tuple(self):
                 "start_date": self.start_date,
                 "report_file": self.report_file,
             }
-            LOG.info(log_json(msg=f"processing OCP on {self.provider_type} to parquet", context=ctx))
             with OCPReportDBAccessor(self.schema_name) as accessor:
                 if not accessor.get_cluster_for_provider(ocp_provider_uuid):
                     LOG.info(
                         log_json(
-                            msg=f"no cluster information available - skipping OCP on {self.provider_type} parquet processing",  # noqa: E501
+                            msg=f"no cluster information available - skipping OCP on {self.provider_type} processing",  # noqa: E501
```

Comment:
Nitpick: you can get rid of the `noqa` directive.

```suggestion
                            msg=f"no cluster information available - skipping OCP on {self.provider_type} processing",
```